File: length-of-last-word/length-of-last-word.cs
public class Solution {
    public int LengthOfLastWord(string s) {
        int letters = 0, i = s.Length; 
        while (s[--i] == ' ') {}                    // Traverse past whitespace
        while (i >= 0 && s[i--] != ' ') letters++;  // Count letters in last word
        return letters;
    }
}

File: length-of-last-word/length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        int letters = 0, i = s.length(); 
        while (s.charAt(--i) == ' ') {}                    // Traverse past whitespace
        while (i >= 0 && s.charAt(i--) != ' ') letters++;  // Count letters in last word
        return letters;
    }
}
